src/Microsoft.Framework.PackageManager/Bundle/BundleManager.cs

```diff
@@ -284,5 +289,48 @@ private DependencyContext CreateDependencyContext(Runtime.Project project, Frame
             dependencyContext.Walk(project.Name, project.Version);
             return dependencyContext;
         }
+
+        private bool ResolveActualRuntimeNames(IList<string> runtimes)
+        {
+            for (int i = 0; i < runtimes.Count(); i++)
+            {
+                if (string.Equals("active", runtimes[i], StringComparison.OrdinalIgnoreCase))
+                {
+                    string activeRuntime = GetActiveRuntimeName();
+                    if (string.IsNullOrEmpty(activeRuntime))
+                    {
+                        _options.Reports.Error.WriteLine("Cannot resolve the active framework name".Red());
+                        return false;
+                    }
+
+                    _options.Reports.Verbose.WriteLine("Resolved the active runtime as {0}", activeRuntime);
+                    runtimes[i] = activeRuntime;
+                }
+            }
+
+            return true;
+        }
+
+        private static string GetActiveRuntimeName()
+        {
+            string pathVariable = Environment.GetEnvironmentVariable("PATH");
+
+            if (!string.IsNullOrEmpty(pathVariable))
+            {
+                string kpmExecutable = PlatformHelper.IsWindows ? "kpm.cmd" : "kpm";
+
+                foreach (string folder in pathVariable.Split(new char[] { Path.PathSeparator }, StringSplitOptions.RemoveEmptyEntries))
+                {
+                    if (File.Exists(Path.Combine(folder, kpmExecutable)) &&
+                        string.Equals("bin", Path.GetFileName(folder)))
```

Comment:
Do we want to prevent failure caused by path with a trailing separator (e.g. `%userprofile%\.k\runtimes\kre-clr-win-x86.1.0.0\bin\`)?

I guess `kvm` guarantees a bin path without trailing separator. However, I think we don't want to be broken silently by `kvm` changes later, or runtime bin path manually added by a user.